pkg/controller/jobs/jobset/jobset_webhook.go

```diff
@@ -57,6 +68,31 @@ func (w *JobSetWebhook) Default(ctx context.Context, obj runtime.Object) error {
 	log.V(5).Info("Applying defaults", "jobset", klog.KObj(jobSet))
 
 	jobframework.ApplyDefaultForSuspend(jobSet, w.manageJobsWithoutQueueName)
+	if features.Enabled(features.MultiKueue) {
+		localQueueName, found := jobSet.Labels[constants.QueueLabel]
+		if !found {
+			return nil
+		}
+		clusterQueueName, err := w.queues.ClusterQueueFromLocalQueue(fmt.Sprintf("%s/%s", jobSet.ObjectMeta.Namespace, localQueueName))
+		if err != nil {
+			return err
+		}
+		clusterQueue, found := w.cache.GetClusterQueue(clusterQueueName)
+		if !found {
+			return nil
+		}
+		for admissionCheckName := range clusterQueue.AdmissionChecks {
+			admissionCheck, ok := w.cache.GetAdmissionCheck(admissionCheckName)
+			if !ok {
+				continue
+			}
+			if admissionCheck.Controller == multikueue.ControllerName {
+				jobSet.Spec.ManagedBy = ptr.To(multikueue.ControllerName)
```

Comment:
This may override a user's custom setting. To be on a safe side I would suggest setting it only if it is nil, or JobSetControllerName.